tests/kernel/fatal/exception/src/main.c

```diff
@@ -79,6 +79,9 @@ void entry_cpu_exception(void *p1, void *p2, void *p3)
 	__asm__ volatile ("trap");
 #elif defined(CONFIG_ARC)
 	__asm__ volatile ("swi");
+#elif defined(CONFIG_RISCV) && defined(CONFIG_XIP)
+	/* illegal instruction on RISCV */
+	__asm__ volatile (".long 0");
 #else
 	/* Triggers usage fault on ARM, illegal instruction on RISCV
```

Comment:
With no `CONFIG_XIP` check, then you need to update here to remove RISCV comment.

Reply:
Done